src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -20,11 +21,16 @@
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_EMAIL = "amy@gmail.com";
     public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
+    public static final String DEFAULT_NOK_NAME = "Amy Bee";
+    public static final String DEFAULT_NOK_PHONE = "85355255";
+    public static final String DEFAULT_NOK_EMAIL = "amy@gmail.com";
+    public static final String DEFAULT_NOK_ADDRESS = "123, Jurong West Ave 6, #08-111";
```

Comment:
Is this a good idea to use the same? possible that some test might pass when it shouldn't?

Reply:
You're right, I'll change this. Thanks for the review!